notebooks/platform/xarray/lib/io/pysnptools_backend.py

```diff
@@ -104,15 +110,36 @@ def read_plink(self, path: PathType, chunks='auto', fam_sep='\t', bim_sep=' ', c
             asarray=False,
             name=_array_name(self.read_plink, path)
         )
-        # pylint: disable=no-member
-        ds = core.create_genotype_count_dataset(arr)
 
-        # Create variant/sample datasets from dataframes
-        ds_fam = dataframe_to_dataset(df_fam, dim='sample', compute_lengths=True, add_coord=False)
-        ds_bim = dataframe_to_dataset(df_bim, dim='variant', compute_lengths=True, add_coord=False)
+        # TODO: either avoid computing Dask arrays, or just use Pandas
+        df_bin_pd = df_bim.compute()
```

Comment:
Hey @tomwhite, some of these dataframes will have 10s of millions of rows (maybe closer to 100M even for PLINK) so I wanted to see why you wanted to move away from chunked arrays here.  Was this just for temporary simplicity or is there something fundamentally very inefficient in the dask.Series to dask.Array conversion?

Reply:
This was just for temporary simplicity to check that the representation was feasible. Not intended to be merged, so feel free to throw away or do what you want with it!

Reply:
Ok makes sense, ty